src/donejs-carousel-plugin.js

```diff
@@ -29,18 +30,32 @@ export const isMobile = function(isTablet) {
 
 export const ViewModel = DefineMap.extend({
 	/**
-	* @property {array} slides populates slides with the appropriate data. This will likely change.
+	* @property {array} slides 
+	* @description populates slides with the appropriate data. This will likely change.
 	*/
 	slides: 'any',
 	/**
-	* @property {number} activeSlideIndex slide that the user wants to see
+	* @property {number} activeSlideIndex 
+	* @description slide that the user wants to see
 	*/
 	activeSlideIndex: {
 		type: 'number',
 		value: 0
 	},
 	/**
-	* @property {number} lastSlideIndex index of last slide in the array
+	* @property {number} activeSlide 
+	* @description active slide dom element
+	*/
+	activeSlide: {
+		get() {
+			if (this.activeSlideIndex !== undefined && this.classSelector) {
+				return $(`${this.classSelector} .slide${this.activeSlideIndex}`);
```

Comment:
active slide can also be referenced by doing `$(``${this.classSelector} .slide.active``);` I think this would obviate the need for you to add the active slide index to the template?

Reply:
yup, exactly